bigchaindb/backend/localmongodb/schema.py

```diff
@@ -99,3 +100,13 @@ def create_metadata_secondary_index(conn, dbname):
 
     # full text search index
     conn.conn[dbname]['metadata'].create_index([('$**', TEXT)], name='text')
+
+
+def create_utxos_secondary_index(conn, dbname):
+    logger.info('Create `utxos` secondary index.')
+
+    conn.conn[dbname]['utxos']\
+        .create_index([
+            ('transaction_id', ASCENDING),
+            ('output_index', ASCENDING),
+        ], name='utxo')
```

Comment:
If I am not mistaken this index is unique? i.e. `unique=True`

Reply:
addressed in a756b482c662c471216b7eb6cd0dd6fa49714eae